server/sonar-server/src/main/java/org/sonar/server/es/EsClient.java

```diff
@@ -193,6 +202,17 @@ public Client nativeClient() {
     return nativeClient;
   }
 
+  /**
+   * Checks whether there is any document in any mentioned type.
+   */
+  public boolean isEmpty(IndexTypeId... indexType) {
+    return count(indexType) <= 0;
+  }
+
+  public long count(IndexTypeId... indexType) {
```

Comment:
"I will need it" is a bad reason. It leads to over-engineered code. I recommend to see the session that JC gave this week about design of APIs.